src/pagination/__tests__/pagination.spec.ts

```diff
@@ -109,4 +109,16 @@ describe('Pagination', () => {
 
     expect(refine).not.toHaveBeenCalled();
   });
+  it('should not add ais-Pagination--noRefinement CSS class on root when nbPages > 1', () => {
+    const fixture = render({ nbPages: 2 });
+    expect(fixture).toMatchSnapshot();
```

Comment:
Instead of relying solely on snapshots, can we add an actual assertion? It's easy to get away from the actual intent.

Reply:
added here: 690adfd